lib/cc/engine/analyzers/python/node.rb

```diff
@@ -3,7 +3,7 @@ module Engine
     module Analyzers
       module Python
         class Node
-          SCRUB_PROPERTIES = ["_type", "attributes"].freeze
```

Comment:
Nope, this is me trying to trim down the resulting s-expressions since they have a heavy mass.